src/common/utils/getOrderApprovalsSetting.js

```diff
@@ -0,0 +1,18 @@
+import { get } from 'lodash';
+// eslint-disable-next-line import/prefer-default-export
+export const getOrderApprovalsSetting = (configs) => {
+  let orderApprovalsSetting = get(configs, [0, 'value'], '{}');
+  const config = {
+    isApprovalRequired: false,
+  };
+
+  try {
+    orderApprovalsSetting = JSON.parse(orderApprovalsSetting);
+  } catch (e) {
+    orderApprovalsSetting = {};
+  }
+
+  Object.assign(config, orderApprovalsSetting);
```

Comment:
I think it's redundant, it's enough to handle it in try catch without default values object